optaplanner-docs/src/main/asciidoc/QuickStart/VaccinationScheduler/VaccinationSchedulerQuickStart-chapter.adoc

```diff
@@ -0,0 +1,275 @@
+. Extract the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-decision-services-quickstarts.zip` file.
+. Navigate to the `optaplanner-quickstarts-{OPTAPLANNER_BOM_VERSION}` directory.
+. Navigate to the `optaplanner-quickstarts-{OPTAPLANNER_BOM_VERSION}/quarkus-vaccination-scheduling` directory.
+endif::OPTAPLANNER-ENT[]
+
+. Enter the following command to start the OptaPlanner vaccination appointment scheduler in development mode:
++
+[source, shell]
+----
+$ mvn quarkus:dev
+----
+
+. To view the OptaPlanner vaccination appointment scheduler, enter the following URL in a web browser.
++
+[source]
+----
+http://localhost:8080/
+----
+
+. To run the OptaPlanner vaccination appointment scheduler, click *Solve*.
+. Make changes to the source code then press the F5 key to refresh your browser. Notice that the changes that you made are now available.
+
+[id="vaccination-scheduler-package-proc_{context}"]
+== Package and run the OptaPlanner vaccination appointment scheduler
+
+When you have completed development work on the OptaPlanner vaccination appointment scheduler in `quarkus:dev` mode, run the application as a conventional JAR file.
+
+.Prerequisites
+
+ifdef::OPTAPLANNER-ENT[]
+* You have downloaded the OptaPlanner vaccination appointment scheduler quick start. For more information, see xref:vaccination-scheduler-download-proc_{context}[].
+endif::OPTAPLANNER-ENT[]
+ifdef::OPTAPLANNER-COMM[]
+* You have downloaded the OptaPlanner vaccination appointment scheduler quick start. For more information, see xref:vaccination-scheduler-download-proc_{context}[].
+endif::OPTAPLANNER-COMM[]
+
+.Procedure
+. Navigate to the `quarkus-vaccination-scheduling` directory.
+. To compile the OptaPlanner vaccination appointment scheduler, enter the following command:
++
+[source, shell]
+----
+$ mvn package
+----
+
+. To run the compiled OptaPlanner vaccination appointment scheduler, enter the following command:
++
+[source, shell]
+----
+$ java -jar ./target/*-runner.jar
+----
++
+[NOTE]
+====
+To run the application on port 8081, add `-Dquarkus.http.port=8081` to the preceding command.
+====
+
+. To start the OptaPlanner vaccination appointment scheduler, enter the following URL in a web browser.
++
+[source]
+----
+http://localhost:8080/
+----
+
+[id="vaccination-native-proc_{context}"]
+
+== Run the OptaPlanner vaccination appointment scheduler as a native executable
+
+To take advantage of the small memory footprint and access speeds that Quarkus offers, compile the OptaPlanner vaccination appointment scheduler in Quarkus native mode.
+
+.Prerequistes.
+
+.Procedure
+
+. Install GraalVM and the `native-image` tool. For information, see https://quarkus.io/guides/building-native-image#configuring-graalvm[Configuring GraalVMl] on the Quarkus website.
+. Navigate to the `quarkus-vaccination-scheduling` directory.
```

Comment:
(soft) This duplicates information from the README
https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/use-cases/vaccination-scheduling/README.adoc#run-a-native-executable
Maybe, that's worth it?